lib/shopify_app/app_proxy_verification.rb

```diff
@@ -3,7 +3,7 @@ module AppProxyVerification
     extend ActiveSupport::Concern
 
     included do
-      skip_before_action :verify_authenticity_token
+      skip_before_action :verify_authenticity_token, raise: false
```

Comment:
Rails 5 raises an exception if you try to act on an action that isn't defined. Easiest compatibility option is to skip raising the error so it acts like Rails 4.